package.json

```diff
@@ -1,64 +1,40 @@
 {
   "name": "react-palm",
-  "version": "0.8.1",
+  "version": "1.0.0",
   "description": "Elm-like architecture for React apps",
   "main": "dist/src",
   "homepage": "https://github.com/btford/react-palm",
   "scripts": {
-    "build": "rm -rf dist && tsc",
-    "build-example": "npm run build && webpack dist/example/main.js dist/example/main.bundle.js",
-    "test": "npm run build && ava",
-    "flow": "flow && node ./scripts/perfect-coverage.js",
-    "watch": "node bin/watch.js"
+    "build":
+      "rm -rf dist && babel src --out-dir dist && flow-copy-source -v src/ dist",
```

Comment:
remove the newline here

Reply:
Should it copy the `package.json` too in order to publish from dist?

Reply:
I'd like to do that in the future, but I'd prefer to change as little as possible for now.

Reply:
0k